erpnext/hr/doctype/attendance/attendance.json

```diff
@@ -172,13 +176,34 @@
    "fieldname": "early_exit",
    "fieldtype": "Check",
```

Comment:
Early exit and late entry should be calculated using In and Out time. Using field `start_time` and `end_time` in Shift type document.

Reply:
Yes, this already happens here: https://github.com/frappe/erpnext/blob/develop/erpnext/hr/doctype/shift_type/shift_type.py#L43